source/mapping-rules/schema-mapping.txt

```diff
@@ -1,3 +1,137 @@
 ==============
 Schema Mapping
-==============
\ No newline at end of file
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Schema mapping is the process by which you decide how your source relational schema 
+should be represented in your target MongoDB database. You facilitate the schema 
```

Comment:
(wording) 
Consider just "Facilitate the schema ..." 
Reasoning is to avoid redundancy.

Reply:
Done.